entrees/vegan-green-bean-hot-dish.md

```diff
@@ -0,0 +1,34 @@
+# Vegan green bean hot dish
+
+A perfect holiday dish, based off vegan recipes from [The Foodie Eats](https://thefoodieeats.com/vegan-green-bean-casserole/)
+
+
+## Ingredients
+
+- 8 ounces baby bella mushrooms, diced
+- 1 tablespoon olive oil
+- 1 teaspoon sea salt
+- Pinch of black pepper
+- 1 teaspoon garlic minced
+- 1 cup soy milk or oat milk
+- 1/2 teaspoon onion powder
+- 1.5 tablespoon cornstarch
+- 20 oz canned French-style green beans drained
+- 4 oz French-fried onions
+
+
+## Instructions
+
+1. In a large sauce pan, add oil, mushrooms, garlic, salt, and pepper. Saute over medium-high heat for 10-15 minutes, until mushrooms have released their juices and liquid has mostly evaporated.
+
+2. In a medium bowl, combine milk, onion powder, and cornstarch. Mix well, making sure there are no lumps.
+
+3. Add milk mixture to mushrooms and reduce to medium-low heat. Cook until desired thickness is reached, probably about 5 minutes.
+
+4. Preheat oven to 350 degrees.
```

Comment:
I would make the preheating step earlier so that it is ready to bake when the ingredients are combined.